greenpithumb/clock.py

```diff
@@ -43,12 +43,20 @@ def __init__(self, clock, duration):
         """
         self._clock = clock
         self._duration = duration
-        self._start_time = self._clock.now()
+        self.reset()
+
+    def set_remaining(self, time_remaining):
+        if time_remaining < datetime.timedelta(seconds=0):
+            raise ValueError('time_remaining must be non-negative')
```

Comment:
Wondering why you decided to add a check for this. I feel like in the past we've just put in the docstring what values are valid and left it at that.